numpy/core/code_generators/generate_umath.py

```diff
@@ -955,6 +955,13 @@ def english_upper(s):
           TD(ints),
           TD('O', f='npy_ObjectLCM'),
           ),
+'bit_count' :
+    Ufunc(1, 1, None,
+          docstrings.get('numpy.core.umath.bit_count'),
+          None,
+          TD(ints),
```

Comment:
This means that the bitcount of `int8` will return `int8` and a bitcount of `uint8` will return `uint8`, etc.  I am wondering if anyone has opinions on this?  It seems reasonable, but right now I am unsure enough to pause.  In theory every bit count can be represented using `uint8` (which would be strange though?), but using the default integer for counting `uint8` bits always also seems a bit strange.  (Basically, I am just not sure that our prior art of homogeneous loops quite applies to bit counting).